content/_components/shopify-admin-v2/index.md

```diff
@@ -529,6 +521,6 @@ There is no configuration fields in this action.
 ## Known limitations
 
 * Look at [Shopify API rate limits](https://shopify.dev/api/usage/rate-limits), specially [GraphQL Admin API rate limits](https://shopify.dev/api/usage/rate-limits#graphql-admin-api-rate-limits)
-* If the component reaches API rate limit it will retry request after waiting until queue will be restored up to 10 times: for example - query cost `500` points, currently available only `100` points, restore rate `50` points/second, component will wait `8` seconds until available points will be restored and try again to get data
+* If the component reaches API rate limit it will retry request after waiting until queue will be restored up to 10 times: for example - query cost `500` points, currently available only `100` points, restore rate `50` points/second, component will wait `8` seconds until available points will be restored and try again to get data.
```

Comment:
* If the component reaches the API rate limit it will retry the request after waiting until the queue is restored up to 10 times: for example - a query costs `500` points, currently available only `100` points, restore rate `50` points/second, component will wait `8` seconds until available points will be restored and try again to get data.